.teamcity/src/main/kotlin/model/CIBuildModel.kt

```diff
@@ -120,12 +120,12 @@ data class CIBuildModel(
                 TestCoverage(7, TestType.parallel, Os.LINUX, JvmCategory.MAX_LTS_VERSION, DEFAULT_LINUX_FUNCTIONAL_TEST_BUCKET_SIZE),
                 TestCoverage(8, TestType.soak, Os.LINUX, JvmCategory.MAX_LTS_VERSION, 1),
                 TestCoverage(9, TestType.soak, Os.WINDOWS, JvmCategory.MIN_VERSION_WINDOWS, 1),
-                TestCoverage(35, TestType.soak, Os.MACOS, JvmCategory.MIN_VERSION, 1),
+                TestCoverage(35, TestType.soak, Os.MACOS, JvmCategory.MIN_VERSION, 1, arch = Arch.AARCH64),
```

Comment:
❔ Why are you passing in architecture here, given that the default should be AARCH64 on Mac?

Reply:
I'd like to make all arches explicit for mac os so people reading this don't need to think about "what's the default arch for our mac builds"?